modules/deser-lib/src/cbor.ts

```diff
@@ -27,7 +27,7 @@ function getType(value: unknown): string {
     );
   }
   if (typeof value === 'string') {
-    if (value.startsWith('0x')) {
+    if ((value as string).startsWith('0x')) {
```

Comment:
the if condition on the line above is no longer a valid type guard?

Reply:
It is - but it actually constrains to `never` here.

I believe a type guard higher up is to blame.